import-core/import-services/src/main/java/org/devgateway/importtool/services/processor/IATI1XProcessor.java

```diff
@@ -400,7 +400,7 @@ private String getFieldName(String fieldName) {
 		String name = fieldName;
 		switch(fieldName) {
 		case "sector":
-			name = "transaction/sector";
+			name = "//sector";
```

Comment:
wouldn't this be mixing up sectors? I mean we could have two different sectors in the activity and like 8 sectors (4 each) in transactions

Reply:
This is only used for filtering for activities. In short what it does is find activities that have a particular sector - when filtering it does not matter if the sector tag is under "transaction" or under iati-activity. It does not affect how we extract data for import.